guides/decoding-blocks-transactions/getBlockById.js

```diff
@@ -0,0 +1,32 @@
+const { apiClient } = require('@liskhq/lisk-client');
+let clientCache;
+const nodeAPI = 'ws://localhost:8080/ws';
+
+const getClient = async () => {
+	if (!clientCache) {
+		clientCache = await apiClient.createWSClient(nodeAPI);
+	}
+	return clientCache;
+};
+
+if (process.argv.length < 3) {
+	console.error("Please provide the block ID to be decoded.")
+	process.exit(1);
+}
+const blockId = process.argv[2];
+
+
+getClient().then((client) => {
+	client.invoke("app:getBlockByID", {
+		id: blockId
+	}).then(res => {
+		console.log(res);
+		const blockObject = client.block.decode(res);
+		const blockJSON = client.block.toJSON(blockObject);
+		console.log(blockJSON);
+		if (blockJSON.payload && blockJSON.payload.length > 0) {
+			console.log(blockJSON.payload[0].asset);
+		}
```

Comment:
Is there a reason why you don't use `blockObject` instead of `blockJSON`? I think `blockJSON` is not necessary here, but please tell if I'm missing something.

Reply:
Yes, I like JSON output more, it turn Buffer into hex string, it is more human readable